app/assets/javascripts/controllers/container_dashboard/util/charts-mixin.js

```diff
@@ -49,6 +83,36 @@ angular.module('miq.util').factory('chartsMixin', function() {
       units    : __('KBps'),
       dataName : __('KBps'),
       tooltipFn  : dailyTimeTooltip
+    },
+    dailyPodUsageConfig: {
+      chartId     : 'podUsageDailyChart',
+      headTitle   : __('Pod Utilization Trends'),
```

Comment:
In the first mockup it was:
_Container Group Trends_
_Image Creation Trends_

is 
_Pod Trends_
_New Image Trends_

maybe _Pod Creation and Deletion Trends_ ?